src/ui/FacetValueSuggestions/FacetValueSuggestions.ts

```diff
@@ -142,9 +142,13 @@ export class FacetValueSuggestions extends Component {
 
   private queryStateFieldFacetId;
 
+  private suggestionHTMLRepresentation: Map<string, string>;
+
   static defaultTemplate(this: FacetValueSuggestions, row: IFacetValueSuggestionRow, omnibox: Omnibox): string {
-    const keyword = DomUtils.highlightElement(row.keyword, omnibox.getText(), 'coveo-omnibox-hightlight2');
-    const facetValue = DomUtils.highlightElement(row.value, row.value, 'coveo-omnibox-hightlight');
+    const keyword =
+      (this.suggestionHTMLRepresentation && this.suggestionHTMLRepresentation[row.keyword]) ||
+      DomUtils.highlightElement(row.keyword, omnibox.getText(), 'coveo-omnibox-hightlight');
```

Comment:
Unit tests happened to test a scenario where suggestions were provided differently, which didn't add a new value to `suggestionHTMLRepresentation`. I added this condition because I was worried that this wouldn't be the only instance of this problem.